src/gui/tray/activitylistmodel.cpp

```diff
@@ -381,6 +390,22 @@ void ActivityListModel::activitiesReceived(const QJsonDocument &json, int status
             a._subjectDisplay = displayString;
         }
 
+        if(Systray::instance()->darkMode()) {
+            auto colorIconPath = QStringLiteral("qrc:///client/theme/white/");
+            if(a._icon.contains("change.svg")) {
+                colorIconPath.append("change.svg");
+                a._icon = colorIconPath;
+            } else if(a._icon.contains("calendar.svg")) {
+                colorIconPath.append("calendar.svg");
+                a._icon = colorIconPath;
+            } else if(a._icon.contains("personal.svg")) {
+                colorIconPath.append("user.svg");
+                a._icon = colorIconPath;
+            }  else if(a._icon.contains("core/img/actions") && Systray::instance()->darkMode()) {
+                a._icon.insert(a._icon.indexOf(".svg"), "-white");
+            }
+        }
```

Comment:
Ideally these should be acquired directly in the correct color from the server API, with a switch for black/white in the activity request